docs/modules/converter/pages/syntax/title.adoc

```diff
@@ -1,16 +1,49 @@
-= Title Slide Customization
-:navtitle: Title Slide
+= Title Slide
 
-The title slide is customized via AsciiDoc attributes.
-These are the global variable assigned at the top of a document under the lead
-title that look like this: `:name: value`.
+The title slide is where customization for the generated presentation can be configured via AsciiDoc xref:asciidoc:attributes:document-attributes.adoc[document attributes].
+These are the document-scope variables assigned at the top of a document, as part of the document header as xref:asciidoc:attributes:attribute-entries.adoc[attribute entries].
 
-This converter supports changing the color, image, video, iframe and
-transitions of the title slide.
+[source,asciidoc]
+----
+= Title Slide
+:revealjs_theme: sky
+:source-highlighter: highlight.js
 
-Read link:{url-revealjs-doc}#slide-backgrounds[the relevant reveal.js documentation] to understand what attributes need to be set.
-Keep in mind that for title slides you must replace `data-` with `title-slide-`.
+== Slide One
 
-See link:{url-project-examples}/title-slide-image.adoc[title-slide-image.adoc].
+* Foo
+* Bar
+* World
+----
 
-The title slide is also added a `title` CSS class to help with template customization.
+In AsciiDoc terms, the first line is the document title and the variable definitions following it are part of the xref:asciidoc:document:header.adoc[document header].
+
+Note that starting your document with a section title means AsciiDoc interprets your Document Header to be empty.
+Any variables set will not be picked up by the converter and will not influence the resulting presentation.
+The next example shows this mistake:
+
+[source,asciidoc]
+----
+== This is not a document title
+// These settings are not interpreted as Document Attributes and will not be accessible to
+// the asciidoctor-revealjs converter.
+:revealjs_theme: sky
+:source-highlighter: highlight.js
+----
+
+NOTE: Even if you start a document with a level-1 section, you can still have a document header.
+The variables then need to be defined _above_ the section header. Another way to think of this is
+having a document header without a document title.
+
+== Customizing The Title Slide
+
+In addition to configuring document wide settings, the Title Slide can also have specialized visual customization that is only applied to the Title Slide.
+
+This converter supports changing the color, image, video, iframe and transitions of the Title Slide.
+
+Read {url-revealjs-doc}#slide-backgrounds[the reveal.js documentation] to understand what attributes can be set.
+Keep in mind that for a Title Slide you must replace `data-` with `title-slide-` in the name of the attribute.
+
+See link:{url-project-examples}/title-slide-image.adoc[title-slide-image.adoc] for an example on using these attributes.
+
+The Title Slide also given an special `title` CSS class to help with template customization.
```

Comment:
This sentence is a bit ambiguous (at least for me). Do we want to say that the Asciidoctor reveal.js converter will automatically add a CSS class named `title` on the title slide?

```suggestion
The title slide also given an special `title` CSS class to help with template customization.
```

Reply:
That is what happens right?

Reply:
I had trouble reading this sentence because I think there's a missing "is". With this addition it reads fine.